controllers/cluster/capabilities.go

```diff
@@ -0,0 +1,42 @@
+package cluster
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+
+	v1 "github.com/openshift/api/config/v1"
+	configv1 "github.com/openshift/client-go/config/clientset/versioned/typed/config/v1"
+)
+
+type Capabilities struct {
```

Comment:
I like the idea of introducing this Capabilties type! But I think we miss the chance to better use it? E.g. what about having a single function here which returns the filled Capabilty struct, and then we use that in main, and  pass it around to where we need it? It could replace the IsOnOpenshift function / var. At the moment it looks like a wild mix of old a new stuff ;) WDYT?

Reply:
I see your point, it makes sense :+1: